features/tools.mdsd.ecoreworkflow.mwe2lib.xtext.feature/feature.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="tools.mdsd.ecoreworkflow.mwe2lib.xtext.feature"
+      label="MDSD.tools EcoreWorkflow MWE2 Library for Xtext"
+      version="0.1.0.qualifier"
+      plugin="org.palladiosimulator.branding">
```

Comment:
Is there a particular reason for using the Palladio branding? MDSD.tools should be an independent project.

Reply:
True, I will remove the branding until we have a MDSD.tools branding.